modules/arkit/arkit_interface.mm

```diff
@@ -286,25 +305,28 @@
 
 		remove_all_anchors();
 
-		[ar_session release];
+		if (@available(iOS 11.0, *)) {
+			[ar_session release];
+			ar_session = NULL;
+		}
 		[ar_delegate release];
-		ar_session = NULL;
+
 		ar_delegate = NULL;
 		initialized = false;
 		session_was_started = false;
 	}
 }
 
 Size2 ARKitInterface::get_render_targetsize() {
-	_THREAD_SAFE_METHOD_
+	//	_THREAD_SAFE_METHOD_
```

Comment:
Why is this needed?

Style wise, it should be:
```suggestion
	//_THREAD_SAFE_METHOD_
```

Reply:
`_THREAD_SAFE_METHOD` is used in old ARKit implementation, but in `master` gives a `Mutex` error on iOS, so I've commented it. 
Will fix styling :)